src/test/java/com/hotels/hcommon/hive/metastore/compatibility/HiveMetaStoreClientCompatibility12xTest.java

```diff
@@ -15,6 +15,8 @@
  */
 package com.hotels.hcommon.hive.metastore.compatibility;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
```

Comment:
Not sure if it makes a difference, but we usually import `org.hamcrest.CoreMatchers.is`

Reply:
Good catch, and indeed the `assertThat` should be `junit`, thanks!

Reply:
Ah yes, I missed that xD